requirements.txt

```diff
@@ -5,8 +5,8 @@ numpy==1.24.2
 notebook==6.5.3
 scikit-learn==1.2.2
 scipy==1.10.1
-tensorflow==2.11.1
-keras==2.11.0
+tensorflow==2.13.0
+keras==2.13.1
```

Comment:
Thanks for updating these deps. One reason it's needed is for people using Python 3.11: tensorflow 2.11.1 has not been released for it. Only tensorflow 2.12 and onward have been release for python 3.11.
`setup.py` has to be updated too, though. As well as LIBRARY_VERSION = '2.0.2' there, to 2.0.3

Reply:
👍